packages/esm-ward-app/src/beds/occupied-bed.component.tsx

```diff
@@ -1,23 +1,18 @@
-import React from 'react';
-import { type Patient } from '@openmrs/esm-framework';
-import { type Bed } from '../types';
-import styles from './occupied-bed.scss';
 import { Tag } from '@carbon/react';
+import React from 'react';
 import { useTranslation } from 'react-i18next';
 import WardPatientCard from '../ward-patient-card/ward-patient-card';
+import { type WardBedProps } from '../ward-view/ward-bed.component';
+import styles from './occupied-bed.scss';
 
-export interface OccupiedBedProps {
-  patients: Patient[];
-  bed: Bed;
-}
-const OccupiedBed: React.FC<OccupiedBedProps> = ({ patients, bed }) => {
+const OccupiedBed: React.FC<WardBedProps> = ({ patientInfos, bed }) => {
```

Comment:
What is `patientInfos`?

Reply:
It's an array of compound object that, as of now, only has the patient and the active visit object. It'll have more things like encounters soon.